### ISSUE TITLE
Barcode not shown up

### ISSUE THREAD

**Issue by KannanKrish** [hjam40/Camera.MAUI#152]
Camara Maui version 1.5.1 QR Code not shown up.

Version 1,4,4 QR Code working fine.

**Comment by bobwhitten** [comment]
yeah, I see the same thing.   I'm using this ONLY to display QR code.  Rolled back to 1.4.4 and see the QR code.

**Comment by mrijntjes** [comment]
> Camara Maui version 1.5.1 QR Code not shown up.

Same for me, and i also had to roll back to version 1.4.4 wat solved the problem for now.

**Comment by melihercan** [comment]
Same here; I can detect barcodes with <= 1.4.4. 
Can this library detect QR codes? It cannot detect this one, even with 1.4.4.
![WhatsApp Image 2024-06-17 at 20 35 59_8812a698](https://github.com/hjam40/Camera.MAUI/assets/10947383/30697770-cc0e-4ccf-81a9-74eb4deee866)

Android Scan QR code can detect the code even from this picture

**Comment by mrijntjes** [comment]
Yes, with 1.5.1 you can detect QR codes. But the configuration is a little bit different as in version 1.4.4. To use barcode detection with Camera.MAUI version 1.5.1 you also have to include the Camera.MAUI.ZXing Nuget package . Then in your sourcecode you have to specify the BarCodeDecoder property of your cameraview as shown in the screenshots and don't forget the Barcodeformat.QR code in PossibleFormats!

![image](https://github.com/hjam40/Camera.MAUI/assets/49949388/59b1033b-c3b8-42d2-a8af-dba95893a696)
![image](https://github.com/hjam40/Camera.MAUI/assets/49949388/9c1571b2-dc3f-4569-8f85-1679c4f3f9b9)

**Comment by melihercan** [comment]
Hi @matthiasrijntjes, thanks for the response.  
I followed the GitHub page instructions as you pointed out above, and all of the configurations are in place. But still, no joy.

I just wanted to ask you this. Camera.MAUI.Test from the master branch should work, right? I gave it a try, and neither a bar code nor a QR code was detected. I have tried both on Android and Windows.

**Comment by mrijntjes** [comment]
Hi @melihercan, i've not yet tested the original Camera.MAUI.Test which is currently based on .net 7.0. I've made a new sample repository for you, with a fresh sample based on .net 8.0 and tested on a Windows laptop with frontcamera. The repository is available at [https://github.com/mrijntjes/CameraMaui151sample](https://github.com/mrijntjes/CameraMaui151sample) with a sample QR code inside issue 1 on [https://github.com/mrijntjes/CameraMaui151sample/issues/1](https://github.com/mrijntjes/CameraMaui151sample/issues/1). If you still experience some problems then I suggest you to open or attribute to a new issue while this issue #152 was made for QRcode generating.

**Comment by melihercan** [comment]
@mrijntjes Thanks for the prompt response. I will give it a try tomorrow and report the results here.
Thanks again.

**Comment by melihercan** [comment]
@mrijntjes I have tested the provided repo both on Android and Windows. 
Android works :)
![WhatsApp Image 2024-06-19 at 23 26 32_24dbe388](https://github.com/hjam40/Camera.MAUI/assets/10947383/6923e328-dbf2-4491-ac7d-080461232fc8)
![WhatsApp Image 2024-06-19 at 23 26 32_a5340716](https://github.com/hjam40/Camera.MAUI/assets/10947383/0496a80c-5e35-46dd-9f78-252a4d4ee2a3)

**Comment by melihercan** [comment]
There is something wrong with the Windows settings. The following errors are observed:
If you can fix it, this will be a great reference app for QR detection. 
Thanks for your hard work.

<img width="882" alt="image" src="https://github.com/hjam40/Camera.MAUI/assets/10947383/43cb69b1-355f-4c58-a8b8-b1ed1db75811">

<img width="880" alt="image" src="https://github.com/hjam40/Camera.MAUI/assets/10947383/0347387b-7b86-493a-a301-cad313841f73">

**Comment by mrijntjes** [comment]
I'm happy to hear that you managed to scan the QR code successfully. The Windows configuration is OK (for reference, I started this repository on another PC to see if it works) but the error message you get refers to an Android component. What usually helps is to close Visual Studio, delete your project /bin and /obj folders completely, and reopen your project. Check if Debug and Windows Machine is selected and try again.

**Comment by Baroonn** [comment]
Had this same issue. It turned out to be my phone camera quality. My phone could scan 2D codes like QRCodes with the version 1.5.1 but it wouldn't detect 1D codes. 

Tried my laptop camera and it worked perfectly.
